lambdas/service/app.py

```diff
@@ -222,44 +222,154 @@ def openapi_raw():
                     body=raw_spec)
 
 
-health_spec = {
-    'responses': {
-        '200': {
-            'description': '200 response',
-            'content': {
-                'application/json': {
-                    'schema': {'$ref': '#/components/schemas/Empty'}
-                }
-            }
-        }
-    },
-    'tags': ['Auxiliary']
+health_up_key = {
+    'up': 'indicates with a boolean value if the service is reachable.\n',
+}
+
+health_service_keys = {
+    'elasticsearch': 'indicates whether service can establish a connection with Elasticsearch, '
+                     'ensuring that indexed documents can be access through Elasticsearch.\n',
+    'api_endpoints': 'shows a list of service API endpoints being requested by service, '
+                     'confirming that a client making a similar request will obtain a OK response.\n',
+    **health_up_key
+}
+
+health_all_keys = {
+    'queues': 'lists queues utilized by indexer to process transactions and index documents into '
+              'Elasticsearch. A queue is denoted as healthy if queue stats can be collected, stats are then '
+              'displayed for each available queue.\n',
+    'other_lambdas': 'represent other server-less applications used by Azul in order to make '
```

Comment:
The varying indention of these text blocks annoys me. Using format_description() and starting the paragraph on a new line will fix this.